static/js/financial_aid/functions.js

```diff
@@ -126,7 +126,7 @@ window.financialAidReview = (function(window, document, $) {
    */
   function initiateSearch() {
     var searchQuery = $("#search-query").val();
-    location = BASE_PATH + searchQuery;
+    location = BASE_PATH + searchQuery; // eslint-disable-line no-global-assign
```

Comment:
Might be clearer to do `window.location = BASE_PATH + searchQuery;`, which would avoid the need for the eslint disable.